python/sqlflow_submitter/xgboost/explain.py

```diff
@@ -46,17 +64,14 @@ def xgb_shap_dataset(datasource,
         selected_cols = db.selected_cols(conn.driver, conn, select)
 
     if transform_fn:
-        column_names = transform_fn.get_column_names()
+        feature_names = transform_fn.get_column_names()
```

Comment:
this line call `get_column_names` but return `feature_names`, do you want to keep the meaning consistent?

Reply:
Great idea! The `transform_fn.get_column_names()` returns feature names. So I would like to rename  `transform_fn.get_column_names()` to be  `transform_fn.get_feature_names()` in the next PR.